docs/source/custom_task.rst

```diff
@@ -1,29 +1,53 @@
 Tutorial: Creating a Custom Task
 ================================
 
-In this tutorial we will go over the process of creating a custom task,
-along with a custom data module.
+In this tutorial we will go over the process of creating a custom :class:`~flash.core.model.Task`,
+along with a custom :class:`~flash.data.data_module.DataModule`.
+
+
+The tutorial objective is to create a ``RegressionTask`` to learn to predict if someone has ``diabetes`` or not.
+The diabetes  (stored as a numpy dataset).
```

Comment:
Is this about the dataset? not a complete sentance.

Reply:
Done !